app/models/sidebar_section.rb

```diff
@@ -26,6 +26,33 @@ class SidebarSection < ActiveRecord::Base
   scope :public_sections, -> { where("public") }
   enum :section_type, { community: 0 }, scopes: false, suffix: true
 
+  def reset_community!
+    ActiveRecord::Base.transaction do
+      self.update!(title: "Community")
+      self.sidebar_section_links.destroy_all
+      community_urls =
+        SidebarUrl::COMMUNITY_SECTION_LINKS.map do |url_data|
+          "('#{url_data[:name]}', '#{url_data[:path]}', '#{url_data[:icon]}', '#{url_data[:segment]}', false, now(), now())"
+        end
+
+      result = DB.query <<~SQL
+      INSERT INTO sidebar_urls(name, value, icon, segment, external, created_at, updated_at)
+      VALUES #{community_urls.join(",")}
+      RETURNING sidebar_urls.id
+      SQL
```

Comment:
I'm not sure if I'm misunderstanding something but I see that we're inserting here but we don't seem to be destroying the "old" sidebar_urls. Should this be the case?

Reply:
They are deleted in after destroy callback https://github.com/discourse/discourse/blob/main/app/models/sidebar_section_link.rb#L22

Not perfect but url this is only linkable type which can be safely deleted

Reply:
Thank you for clarifying. This sounds OK for now.